src/sakshi/Assignment23.java

```diff
@@ -0,0 +1,34 @@
+package sakshi;
+
+public class Assignment23 {
+
+	void getCharCaseCount(String str){
+		int uCount=0;
+		int lCount=0;
+		int dCount=0;
+		int spCount=0;
+		
+		for(int index=0; index<str.length(); index++){
+			if(Character.isUpperCase(str.charAt(index))){
```

Comment:
you can write char ch = str.charAt(index) outside of conditional statement, so that you no need to rewrite again and again. you just need to write like Character.isUpperCase(ch)